google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryStorageTest.java

```diff
@@ -873,6 +883,16 @@ public void testStructAndArraySqlTypes() throws InterruptedException, IOExceptio
         Schema.Type.STRING,
         structSchema.getField("str_field").schema().getType());
     assertEquals(rowAssertMessage, new Utf8("abc"), structRecord.get("str_field"));
+
+    // Validate the projected STRUCT field and its members.
+    Schema unionSchema = avroSchema.getField("struct_field_opt").schema();
+    Schema structOpSchema = unionSchema.getTypes().get(1);
+    assertEquals(actualSchemaMessage, 1, structOpSchema.getFields().size());
+    assertEquals(
+            actualSchemaMessage,
+            Schema.Type.LONG,
+            structSchema.getField("int_field").schema().getType());
+    assertNull(rowAssertMessage, record.get("struct_field_opt"));
```

Comment:
Test fails here. whole record is `null` but returned value is `{"int_field": 0}`